proto/account.proto

```diff
@@ -3,31 +3,46 @@ package account;
 
 import "digest.proto";
 
+// An account ID.
 message AccountId {
     // A miden account is defined with a little bit of proof-of-work, the id itself is defined as
     // the first word of a hash digest. For this reason account ids can be considered as random
     // values, because of that the encoding below uses fixed 64 bits, instead of zig-zag encoding.
     fixed64 id = 1;
 }
 
+// A summary of an account.
 message AccountSummary {
+    // The account ID.
     AccountId account_id = 1;
+
+    // The latest account hash, zero hash if the account doesn't exist.
     digest.Digest account_hash = 2;
+
+    // Merkle path to verify the account's inclusion in the MMR.
     uint32 block_num = 3;
 }
 
+// An account info.
 message AccountInfo {
+    // Account summary.
     AccountSummary summary = 1;
+
+    // Account details encoded using Miden native format.
     optional bytes details = 2;
```

Comment:
I would describe what the details actually contain (i think they contain full account state description, right?). Also, we should say why the are optional.